fpdf/html.py

```diff
@@ -263,6 +267,7 @@ def width2unit(self, length):
         return int(length)
 
     def handle_data(self, data):
+        trailing_space = re.search("\s$",data)
```

Comment:
Now that everything seems to work :+1:: In the name of efficiency, you might want to precompile all RE patterns (with `re.compile()`), eg. as class attributes. They're not particularly big or complex, but this will still eliminate a bit of processing in every single use.